controllers/apply-next/lib/formatters.js

```diff
@@ -70,6 +70,19 @@ function formatDate(value) {
     return dt.isValid() ? dt.format('D MMMM, YYYY') : '';
 }
 
+function formatDateRange(value) {
+    const dates = {
+        start: fromDateParts(value.start),
+        end: fromDateParts(value.end)
+    };
+    if (!dates.start.isValid() && !dates.end.isValid()) {
+        return '';
+    }
+    return `${dates.start.format('D MMMM, YYYY')} – ${dates.end.format(
```

Comment:
😍  Fantastic. :neckbeard: I'd typically set the en-dash in date ranges with no space not sure if that's typical or not (yes I have opinions on the use of dashes but I'll leave typos littered across the code-base, thanks for fixing those too!)

Reply:
Done!
![image](https://user-images.githubusercontent.com/394376/58325117-b8715880-7e20-11e9-898b-756a383ae4f8.png)